Classes/Plugin/PageGrid.php

```diff
@@ -91,7 +91,7 @@ protected function getPageBrowser() {
         if ($this->piVars['pointer'] > 0) {
             $output = $this->pi_linkTP_keepPIvars($this->pi_getLL('prevPage', '&lt;', TRUE), ['pointer' => $this->piVars['pointer'] - 1, 'page' => (($this->piVars['pointer'] - 1) * $this->conf['limit']) + 1], TRUE).$separator;
         } else {
-            $output = $this->pi_getLL('prevPage', '&lt;', TRUE).$separator;
+            $output = '<span class="first-page not-active">'.$this->pi_getLL('prevPage', '&lt;', TRUE).'</span>'.$separator;
```

Comment:
Shouldn't this be something like `prev-page` instead of `first-page`?

Reply:
you are right, my mistake!